src/hooks/project-hooks.tsx

```diff
@@ -244,6 +244,9 @@ export const ProjectProvider = ({
       });
     } catch (e) {
       if (e instanceof CodeEditorError) {
```

Comment:
I feel like there's more than one of these cases. Is there a reason to only log here? Can we log in one place and still get all the cases?

Reply:
There's one where land directly on the code page which is a different scenario, but yes, I missed the save hex case.